dev-python/cython/python3_cython-0.25.2.recipe

```diff
@@ -0,0 +1,51 @@
+SUMMARY="Optimising static compiler for Python"
+DESCRIPTION="Cython is an optimising static compiler for both the Python \
+programming language and the extended Cython programming language (based on \
+Pyrex). It makes writing C extensions for Python as easy as Python itself."
+HOMEPAGE="http://cython.org/"
+COPYRIGHT="2007-2016 Stefan Behnel, Robert Bradshaw, et al."
+LICENSE="Apache v2"
+REVISION="1"
+SOURCE_URI="https://pypi.python.org/packages/b7/67/7e2a817f9e9c773ee3995c1e15204f5d01c8da71882016cac10342ef031b/Cython-0.25.2.tar.gz"
+CHECKSUM_SHA256="f141d1f9c27a07b5a93f7dc5339472067e2d7140d1c5a9e20112a5665ca60306"
+SOURCE_DIR="Cython-$portVersion"
+
+ARCHITECTURES="?x86_gcc2 x86 x86_64"
+
+PROVIDES="
+	cython = $portVersion
+	cmd:cygdb
+	cmd:cython
+	cmd:cythonize
```

Comment:
for the python3 package:
for file in cygdb cython cythonize; do mv $binDir/$file $binDir/${file}3 ; done